ADAL/src/telemetry/ADTelemetryDefaultEvent.m

```diff
@@ -130,6 +131,14 @@ - (NSString*)getStringFromDate:(NSDate*)date
     return [s_dateFormatter stringFromDate:date];
 }
 
+- (void)processEvent:(NSMutableDictionary*)eventToBeDispatched
```

Comment:
processEvent: is too generic of a name and doesn't accurately reflect what it's doing.